docs/make.jl

```diff
@@ -68,6 +68,7 @@ pages = [
     "Introduction" => "index.md",
     "Tutorial" => "tutorial.md",
     "Examples" => [
+        "Sugarscape" => "examples/sugarscape.md",

```

Comment:
Because schelling is the example that we use to demonstrate our tutorial, can we keep the Schelling example as the topmost on this list?

Reply:
Yeah, I thought since Schelling is already covered in the Tutorial, we can put a more important model on the top.